### PR TITLE
[PAY-261] Refactor tipping slice amount type from BNWei to StringAudio

### PR DESCRIPTION
**Pull request by dylanjeffers**
### Description

Changes tipping slice amount type from BNWei to StringAudio, which fixes case where `dispatchWeb` incorrectly serializes BNWei (by calling .toString()), resulting in different mobile vs web behavior. Moving to StringWei helps since that is directly serializable across apps.

Also changes StringWei type to be a string alias since it's unclear how these types are actually `string & {_brand: string}` maybe in the future we can change the stringaudio data to properly match these types though.

### Dragons

Changing the underlying type may have messed up the tipping flow, so lets def check this works before merging

### How Has This Been Tested?

Types pass, but a local run-through is needed

### CODE REVIEW

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/dj-pay-261-refactor-tipping-amount-type

File: packages/web/src/components/tipping/tip-audio/TipSent.tsx [hunk 1]
```diff
@@ -31,15 +31,9 @@ export const TipSent = () => {
 
   const handleShareClick = useCallback(() => {
     if (account && recipient) {
-      let recipientAndAmount = `${recipient.name} ${formatWei(
-        sendAmount,
-        true
-      )}`
+      let recipientAndAmount = `${recipient.name} ${formatWei}`
```
Comment (sddioulde) [inline packages/web/src/components/tipping/tip-audio/TipSent.tsx:34]: i think you meant ${sendAmount} instead of the function ${formatWei}

Comment (sddioulde) [reply packages/web/src/components/tipping/tip-audio/TipSent.tsx:34]: plus, we should format the sendAmount itself

**Review by sddioulde — COMMENTED** [review]
looks good, pending comment. i also think we should handle formatting in this PR if possible

**Review by sddioulde — APPROVED** [review]

**Pull request merged**